### PR TITLE
[14.0][FIX] account_banking_sepa_direct_debit: Fix crash caused by typo in print_report_name

### PR DESCRIPTION
**Pull request by alexis-via**
And one small improvement: add print button on mandate form view.

### CODE REVIEW

File: account_banking_sepa_direct_debit/views/account_banking_mandate_view.xml [hunk 1]
```diff
@@ -12,6 +12,15 @@
         <field name="model">account.banking.mandate</field>
         <field name="inherit_id" ref="account_banking_mandate.view_mandate_form" />
         <field name="arch" type="xml">
+            <button name="validate" position="before">
+                <button
+                    name="print_report"
+                    type="object"
+                    string="Print"
+                    class="btn-default"
+                    states="draft,valid"
```
Comment (pedrobaeza) [inline account_banking_sepa_direct_debit/views/account_banking_mandate_view.xml]: Once valid, it should be shown IMO, or at least not as highlighted.

Comment (alexis-via) [reply account_banking_sepa_direct_debit/views/account_banking_mandate_view.xml]: ok, I agree. I removed states=""

Comment (pedrobaeza) [reply account_banking_sepa_direct_debit/views/account_banking_mandate_view.xml]: Agh, I made a typo, I wanted to say `shouldn't be shown`, but well if you put it without states, but not highlighted, it's OK.

Comment (pedrobaeza) [reply account_banking_sepa_direct_debit/views/account_banking_mandate_view.xml]: Please remove then `class="btn-default"`

Comment (alexis-via) [reply account_banking_sepa_direct_debit/views/account_banking_mandate_view.xml]: btn-default is NOT highlighted (it's highligthed when you have btn-primary).

Comment (pedrobaeza) [reply account_banking_sepa_direct_debit/views/account_banking_mandate_view.xml]: Ah, OK, sorry for the confusion.

**Review by pedrobaeza — APPROVED** [review]

**Review by HaraldPanten — APPROVED** [review]

**Pull request merged**